README.md

```diff
@@ -36,8 +36,25 @@ We designed it to acquire data from multiple instruments over timescales of the
 
 # Installation #
 
-For the time being this package is only supported in python 2.7 (but we are working to make it work in python 3)
+This package is designed to work with Python 3. Fresh installations require Python >= 3.6
 
+## Using the setup script
+
+For a fresh install of LabGUI with Python3 and git in the directory where you wish to install, type the following into your favorite shell:
+```
+git clone https://github.com/LabGUI/LabGUI.git
+cd LabGUI
+python setup.py install
+```
+This will clone LabGUI to the current directory using git, set up a virtual environment, install all required packages, and create a configuration file via `setup_new.py` script, and will add LabGUI to the path.
+
+By cloning the repository, you can easily use git to update LabGUI by calling
```

Comment:
What do you mean here?

Reply:
I mean that setup.py actually executes setup_new.py as a sub-process, which can execute config_generate.py as a sub-process should the user wish. (user interactive script)